pyproject.toml

```diff
@@ -101,6 +101,85 @@ build = [
     "setuptools==69.5.1",
     "build==1.2.1"
 ]
+testing = [
+    "deptry>=0.14"
+]
+
+[tool.deptry]
+extend_exclude = [
+    "doc",
+    "scripts",
+    "sigs",
+    "tests"
```

Comment:
let's sort these lists alphabetically or by length?

Reply:
I initially sorted everything alphabetically, but length also makes sense (easily readable).

Reply:
a-z is good, todo for the below lists